GCTC-NTGC/gc-digital-talent
# Admin - Remove CMO Assets UI

Resolves #4983.

## Steps to test

1. Compile admin `npm run intl-compile --workspace=admin`
2. Build admin `npm run production --workspace=admin`
3. Verify admin app still works without removed code
